MidasCivil_Adapter/Convert/ToMidasCivil/Elements/FromFEMesh.cs

```diff
@@ -24,18 +24,24 @@
 using BH.Engine.Adapter;
 using BH.oM.Structure.Elements;
 using System.Collections.Generic;
+using System;
+using BH.Adapter.MidasCivil;
+using System.Text;
+
+
 namespace BH.Adapter.Adapters.MidasCivil
 {
     public static partial class Convert
     {
         /***************************************************/
         /**** Public Methods                            ****/
         /***************************************************/
-
-        public static string FromFEMesh(this FEMesh feMesh)
+        private static int index = 0;
```

Comment:
Remove this variable as this will not work when we push to a model with existing elements - see changes to `Create` method for `FEMesh`.